docs/docs/features/state-file.md

```diff
@@ -5,52 +5,89 @@ icon: material/file
 
 ## Introduction
 
-The state file is the serialized state of a recce instance. There are two scenarios where we need to export the state file:
+The state file represents the serialized state of a Recce instance. It is a JSON-formatted file containing the following information:
 
-- [PR Review](#pr-review): We can include the state file in the PR Review Comment. If the reviewer wants to further connect to the PR environment, they can use this file to see the final results and perform deeper audits.
-- [Development](#development): During development, we often need to save the state and even switch between different branches for development.
+- **Checks**: Data from the checks added to the checklist on the Checklist page.
+- **Runs**: Each diff execution in Recce corresponds to a run, similar to a query in a data warehouse. Typically, a single run submits a series of queries to the warehouse and retrieves the final results.
+- **Environment Artifacts**: Includes `manifest.json` and `catalog.json` files for both the base and current environments.
+- **Runtime Information**: Metadata such as Git branch details and pull request (PR) information from the CI runner.
 
-## PR Review
+## How to Save the State File
 
-In the PR review process, the state file can serve as a medium of communication between the submitter and the reviewer.
```

Comment:
we use "author" in our articles.

Reply:
oh, we currently use "submitter" in this doc. We can keep it now and then discuss if we want to align the language.

Reply:
Ok